src/jsc/jsc_value.hpp

```diff
@@ -177,8 +177,7 @@ inline double jsc::Value::to_number(JSContextRef ctx, const JSValueRef &value) {
         throw jsc::Exception(ctx, exception);
     }
     if (isnan(number)) {
-        throw std::invalid_argument(util::format("Value '%1' not convertible to a number.",
-                                                 (std::string)to_string(ctx, value)));
+        std::nan("");
```

Comment:
return?

Reply:
Isn't `if (isnan(number) { return std::nan(""); } return number;` just a roundabout way of writing `return number`? When does the extra check do anything?

Reply:
I think you are right. But `NaN` can be represented by multiple bit patterns in IEEE 754. Maybe I am just seeing ghosts where no ghost exist.